### ISSUE TITLE
CrossLet freezing browser

### ISSUE THREAD

**Issue by degmsb** [ramnathv/rMaps#77]
I am running the CrossLet code below and it is freezing my browser.  what is the best way to troubleshoot?

library(rMaps)
crosslet(
  x = "country", 
  y = c("web_index", "universal_access", "impact_empowerment", "freedom_openness"),
  data = web_index
)

**Comment by skishchampi** [comment]
same here @kykuo

**Comment by skishchampi** [comment]
![crosslet403](https://cloud.githubusercontent.com/assets/996985/5947073/31e36278-a700-11e4-9f8f-f2d1550a0420.PNG)

**Comment by mrhunni** [comment]
Yes, I don't think it is about "freezing" the browser. Isn't this error of not displaying caused due to an scritp?

**Comment by ramnathv** [comment]
The `crosslet` function has been very brittle. So I went ahead and started rewriting it from scratch. Hope to post an updated, more solid binding soon. Folks on this thread can probably help me beta test it.

**Comment by skishchampi** [comment]
@mrhunni Like @ramnathv mentioned earlier, the key to the cloudmade api was hardcoded. The 403 response is due to using ramnathv's key being used by so many of us which lead to throttling issues. I am not sure.

**Comment by mrhunni** [comment]
Thank you @skishchampi.

Yeah, it'd be great testing it.

**Comment by cantgetthatping** [comment]
My pleasure to test it
